src/black/mode.py

```diff
@@ -177,6 +177,7 @@ class Preview(Enum):
     wrap_long_dict_values_in_parens = auto()
     multiline_string_handling = auto()
     typed_params_trailing_comma = auto()
+    is_simple_lookup_for_doublestar_expression = auto()
```

Comment:
Could you add this feature to the documentation in `future-style.md`, where we now list all preview features? The description can just be something like "fix line length computation for certain expressions that involve the power operator".

Reply:
Sure thing, I'll add it!